master.tf

```diff
@@ -23,29 +23,37 @@ resource "linode_instance" "k8s_master" {
     kernel = "linode/direct-disk"
 
     devices {
-      sda = {
+      sda {
         disk_label = "boot"
       }
     }
   }
 
   provisioner "file" {
-    source      = "${path.module}/scripts/"
+    source      = "${path.cwd}/${path.module}/scripts/"
     destination = "/tmp"
 
     connection {
-      user    = "core"
-      timeout = "300s"
+      host        = self.ip_address
+      agent       = "false"
+      private_key = chomp(file(var.ssh_private_key))
```

Comment:
Also please remove this var

Reply:
Same deal as the host line -- this must be specified in the connection block for TF 0.12.

Reply:
https://www.terraform.io/upgrade-guides/0-12.html#default-settings-in-connection-blocks

Reply:
Actually -- I am a bit mistaken here:
When agent = "false", a private_key must be specified.

When agent = "true" it is not necessary... so this can be nixed.

The private_key variable was defined in the variables.tf file before I made this PR. Since that var is not used, it can be safely removed from variables.tf as well.